lib/modules/apostrophe-pieces-headless/index.js

```diff
@@ -154,6 +242,15 @@ module.exports = {
 
     };
 
+    var superAllowedSchema = self.allowedSchema;
+    self.allowedSchema = function(req) {
+      var schema = superAllowedSchema(req);
+      if (req.restApiPatchSchema) {
+        schema = _.intersectionBy(schema, req.restApiPatchSchema, 'name');
```

Comment:
Point for improvement: if some must be dropped, that should probably be a 500 error or at least some kind of warning to the dev, but quietly ignoring those fields is adequate